### ISSUE TITLE
[Question] Example with json lint ?

### ISSUE THREAD

**Issue by hellomocks** [logue/vue-codemirror6#4]
Was trying to find any examples with json lint but in vain, can someone help ?

**Comment by logue** [comment]
There is a bug on the VueCodemirror6 side and it cannot be specified with the `lint` prop, but it can be avoided with the `extensions` prop.

For JSON, specify the extensions prop as shown below.

```vue
<code-mirror v-model="value" :extensions="cmExtensions" />
```

```js
    const cmExtensions: Extension[] = [
      linter(jsonParseLinter()),
      lintGutter(),
      basicSetup,
      json(),
    ];
```

The point is not to use the `basic` prop, but to specify it directly in the `extensions` prop.

**Comment by logue** [comment]
Fixed in `0.5.1`.
